RadixWallet/Features/DappInteractionFeature/Interactor/DappInteractor.swift

```diff
@@ -156,18 +126,22 @@ struct DappInteractor: Sendable, FeatureReducer {
 			return .run { _ in
 				_ = await radixConnectClient.loadP2PLinksAndConnectAll()
 			}
+
+		case .completionDismissed:
+			npsSurveyClient.incrementTransactionCompleteCounter()
```

Comment:
Note that just `completionDismissed` is not enough, you need to also check that the completion is for a transaction, and not any other interaction(like Login).

You could potentially make use of the presence of `txId` from the completion state.